ProcessLib/UniformDirichletBoundaryCondition.h

```diff
@@ -67,16 +69,13 @@ class UniformDirichletBoundaryCondition
                                 MeshLib::MeshItemType::Node,
                                 id);
             // TODO: that might be slow, but only done once
-            id = dof_table.getGlobalIndex(l, component_id);
+            const auto g_idx = dof_table.getGlobalIndex(l, component_id);
+            if (g_idx >= 0)
+            {
+                global_ids.emplace_back(g_idx);
```

Comment:
The thing is that we have to count how many non-ghost entries in advance in order to call reserve().